client/components/app.jsx

```diff
@@ -0,0 +1,532 @@
+import React from 'react';
```

Comment:
Splitting App.jsx into three different components may increase readability and follows the principle of separation of concerns. Having components that are dedicated to each type of filter would be the categories that I suggest splitting them on. Obviously MVP first then refactor as needed. Besides that, I believe that this is ready for production.